Tests/XCDiffCoreTests/Library/StringExtensionsTests.swift

```diff
@@ -0,0 +1,74 @@
+//
+// Copyright 2019 Bloomberg Finance L.P.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+@testable import XCDiffCore
+import XCTest
+
+final class StringExtensionsTests: XCTestCase {
+    private var delimiter: String!
```

Comment:
Maybe we can just use "::" whenever we need it? Would make the class shorter and easier to read (hopefully). Then we would need `setUp` method at all.

What do you think about the following? 

```swift
func testSplit_whenEmptyString() {
    // When / Then
    equal("".split(around: "::"), ("", nil))
}
```

Reply:
Yes makes sense. The setup is pointless in that case.